api/OpenAI.net8.0.cs

```diff
@@ -5972,6 +6072,85 @@ public class ResponseReasoningOptions : IJsonModel<ResponseReasoningOptions>, IP
         public override readonly string ToString();
     }
     [Experimental("OPENAI001")]
+    public class ResponseResult : IJsonModel<ResponseResult>, IPersistableModel<ResponseResult> {
+        public DateTimeOffset CreatedAt { get; }
+        public string EndUserId { get; }
+        public ResponseError Error { get; }
+        public string Id { get; }
+        public ResponseIncompleteStatusDetails IncompleteStatusDetails { get; }
+        public string Instructions { get; }
+        public ModelIdsResponses? InternalModel { get; }
```

Comment:
We should rename this one to "Model". Historically, we also don't use an enum for model names, and instead we keep it just as a string.